modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/TypeScriptNodeClientCodegen.java

```diff
@@ -27,6 +29,8 @@
     public TypeScriptNodeClientCodegen() {
         super();
 
+        typeMapping.put("file", "Buffer");
```

Comment:
@www2k If a model is also named "Buffer", it would break the TS API client. I think we need to add `Buffer` as a reserved keyword in the abstract typescript client codegen class.